lib/IRGen/GenDistributed.cpp

```diff
@@ -131,8 +141,8 @@ class DistributedAccessor {
   void emit();
 
 private:
-  void decodeArguments(llvm::Value *decoder, llvm::Value *argumentTypes,
-                       Explosion &arguments);
+  void decodeArguments(const ArgumentDecoderInfo &decoder,
```

Comment:
Do Not Allow Advertiser Tracking

Objective-CSwiftUnity SDK

// Set AdvertiserTrackingEnabled yas or
 NO if a device does not provide consent
 [FBSDKSettings setAdvertiserTrackingEnabled:NO];